src/cryptoadvance/specter/server_endpoints/wallets.py

```diff
@@ -612,6 +616,12 @@ def send_new(wallet_alias):
                 specter=app.specter,
                 rand=rand,
             )
+        elif action == "fillform":
+            # can be used to recommend a transaction from a service (goind to an exchange or so)
+            addresses = request.form.getlist("addresses[]")
+            labels = request.form.getlist("labels[]")
+            amounts = request.form.getlist("amounts[]")
+            fillform = True
```

Comment:
@k9ert, I'm unclear how this was meant to be used. Is it ever actually exposed/used in the UI or more of a future feature?